ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -52,6 +52,8 @@ index.search.slowlog.threshold.query.info: 5s
 index.search.slowlog.threshold.query.debug: 2s
 index.search.slowlog.threshold.query.trace: 500ms
 
+index.refresh_interval: 5s
```

Comment:
thanks, didn't know we did that in the django code. yes i did think about doing it for all indices. i don't really see a downside in increasing it for every index since we don't use it for anything real time. but i suppose in general it'd be best to just do it for forms and cases. i could imagine a giant user upload doing something similar though (or giant ucr insert)